### DISCUSSION TITLE
Maybe a bug - ui.stats() scroll bar

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1368] [General]
broswer: Safari Version 15.2
Wave version 0.21.0

Issue: `ui.stats()` rendering a scroll bar. Example:

![image](https://user-images.githubusercontent.com/64787868/164319922-2b754ee3-345b-4ffb-8664-5f8284692971.png)

Example 2:

<img width="622" alt="image" src="https://user-images.githubusercontent.com/64787868/164320394-40951a07-7158-49a9-8e47-92a24447ae80.png">

**Cannot** reproduce this in wave 0.20.0

**Can** partially reproduce this in wave v0.21.0 in Firefox.
By partially, I mean that the scroll bars are rendered and then immediately disappear.

**Comment by mturoci** [comment]
Can you please provide a min repro code?

**Comment by azim-b** [comment]
@mturoci 

here's another example. Note that setting `inset=True` prevents this redundant scroll bar.

UI examples:

![image](https://user-images.githubusercontent.com/64787868/164478914-4084f0c7-3cd4-4a05-9480-0c4e1661baf0.png)



Repro code:


```py
q.page['body'] = ui.form_card(box='body',items=[
        # without inset
        ui.separator('Without inset'),
        ui.stats(justify='center',items=[
            ui.stat(label='H2O AI Health', value='AI4Good', icon='HealthSolid', icon_color='Red'),
        ]),
        # with inset set
        ui.separator('With inset'),
        ui.stats(justify='center', inset=True, items=[
            ui.stat(label='H2O AI Health', value='AI4Good', icon='HealthSolid', icon_color='Red'),
        ])
    ])
```

**Reply by mturoci** [reply]
Perfect, that's a bug. Thanks for reporting.
